setup.py

```diff
@@ -171,6 +230,63 @@ def get_extensions():
             )
         )
 
+    # ------------------- Torchvision extra extensions ------------------------
+    vision_include = os.environ.get('TORCHVISION_INCLUDE', None)
+    vision_library = os.environ.get('TORCHVISION_LIBRARY', None)
+    vision_include = (vision_include.split(os.pathsep)
+                      if vision_include is not None else [])
+    vision_library = (vision_library.split(os.pathsep)
+                      if vision_library is not None else [])
+    include_dirs += vision_include
+    library_dirs = vision_library
+
+    # Image reading extension
+    image_macros = []
+    image_include = [extensions_dir]
+    image_library = []
+    image_link_flags = []
+
+    # Locating libPNG
+    libpng = distutils.spawn.find_executable('libpng-config')
```

Comment:
I tried this locally on a Windows box after doing `conda install libpng` and it returns None.

Reply:
I suggest changing this to `pngfix`. as it is more robust and is listed in the official page. http://www.linuxfromscratch.org/blfs/view/svn/general/libpng.html

Reply:
```suggestion
libpng = distutils.spawn.find_executable('libpng-config') or distutils.spawn.find_executable('pngfix')
```